src/renderer/prefComponents/markdown/index.vue

```diff
@@ -60,6 +60,15 @@
       :onChange="value => onSelectChange('footnote', value)"
       more="https://pandoc.org/MANUAL.html#footnotes"
     ></bool>
+    <separator></separator>
+    <h5>Style</h5>
```

Comment:
I prefer change `Style` to `Diagram theme`？ Because we maybe also support `mermaid` etc themes.